test/1_stdlib/Inputs/ProcessStressTest/ProcessStressTest.swift

```diff
@@ -0,0 +1,53 @@
+// Do not change the SIL name for this without also changing ProcessStressTest.c
+@_silgen_name("swift_process_test_getProcessArgs")
+public func runTest() {
+  let ProcessRaceTestSuite = TestSuite("Process Race")
+
+  ProcessRaceTestSuite.test("passes") {
+    runRaceTest(ProcessRace.self, trials: 100)
```

Comment:
I think we only need 1 trial here.  Once variables are initialized, there is no second chance for this process to trigger a race.